### PR TITLE
Increase timeouts for herbstclient termination

### PR DESCRIPTION
**Pull request by t-wissmann**
With more clang analyzer checks, it seems that the timeout does not
suffice anymore...

### CODE REVIEW

**Comment by blarz** [comment]
> With more clang analyzer checks, it seems that the timeout does not
> suffice anymore...

Does this really depend on the clang-tidy checks? Is the load too high during tests because of it?

File: tests/test_herbstclient.py [hunk 1]
```diff
@@ -85,7 +85,7 @@ def test_herbstclient_wait(hlwm, num_hooks_sent, num_hooks_recv):
         hlwm.call('emit_hook nonmatch nonarg')
         hlwm.call('emit_hook matcher somearg')
 
-    proc.wait(3)
+    proc.wait(5)
```
Comment (blarz) [inline tests/test_herbstclient.py:88]: Should we extract the timeout to a variable used in both tests?

**Comment by t-wissmann** [comment]
> > With more clang analyzer checks, it seems that the timeout does not
> > suffice anymore...
> 
> Does this really depend on the clang-tidy checks? Is the load too high during tests because of it?

I don't know. Aftere the clang-tidy checks were introduced, the timeout was reached by the following two commits, but it didn't appear again afterwards.

Maybe we can dismiss this PR and see whether the issues shows up again..

File: tests/test_herbstclient.py (hunk 1 above)
Comment (t-wissmann) [reply tests/test_herbstclient.py:88]: not sure whether it's an improvement

Comment (blarz) [reply tests/test_herbstclient.py:88]: It's not that important to me, either ;)

**Comment by blarz** [comment]
> > > With more clang analyzer checks, it seems that the timeout does not
> > > suffice anymore...
> > 
> > 
> > Does this really depend on the clang-tidy checks? Is the load too high during tests because of it?
> 
> I don't know. Aftere the clang-tidy checks were introduced, the timeout was reached by the following two commits, but it didn't appear again afterwards.
> 
> Maybe we can dismiss this PR and see whether the issues shows up again..

Let's see if it happens again, I guess the number of PRs will stay high in the following days ;)

**Pull request merged**